src/pages/patientView/PatientViewPage.tsx

```diff
@@ -115,22 +112,34 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         //TODO: this should be done by a module so that it can be reused on other pages
         const qs = queryString.parse((window as any).location.search);
 
+        const reaction1 = reaction(
+            () => props.routing.query,
+            query => {
 
-        patientViewPageStore.studyId = qs['cancer_study_id'] + '';
+                if ('studyId' in query) {
+                    patientViewPageStore.studyId = query.studyId;
+                } else {
+                    alert("You must have a study Id");
```

Comment:
Yes, for sure.  This is just a placeholder.  I want to have a global error handling/messaging solution.